packages/quick_blue/lib/src/quick_blue_linux.dart

```diff
@@ -187,6 +187,10 @@ class QuickBlueLinux extends QuickBluePlatform {
     // TODO: implement requestMtu
     throw UnimplementedError();
   }
+  
+  @override
+  // TODO: implement availabilityChangeStream
+  Stream get availabilityChangeStream => throw UnimplementedError();
```

Comment:
Move `_availabilityChangeStream` above `startScan` and below `isBluetoothAvailable`